Formula/clisp.rb

```diff
@@ -59,29 +44,27 @@ def install
         s.change_make_var! "CFLAGS", "#{s.get_make_var("CFLAGS")} #{ENV["CFLAGS"]}"
```

Comment:
do we know if this is still necessary? I'm not sure how to check this myself

Reply:
I’m pretty sure that’s also irrelevant now because the superenv automatically strips all -O flags and replaces them with -Os on macOS and -O2 on Linux.

Reply:
awesome, let's see if the build passes without it